### PR TITLE
Practice session 4 4.2

### PR DESCRIPTION
**Pull request by JuanaRodriguez**

### CODE REVIEW

File: Juana/Cucumber_test/features/practice4.3.feature [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Feature: Gmail account
+
+Scenario: Add gmail account first part
+
+Give I have a name, accepting lower case
```
Comment (karimeSalomon) [inline Juana/Cucumber_test/features/practice4.3.feature]: Given

File: Juana/Cucumber_test/features/practice4.3.feature [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+And I have a lastname, accepting lower case
```
Comment (karimeSalomon) [inline Juana/Cucumber_test/features/practice4.3.feature]: Always add indent to you And sentences

File: Juana/Cucumber_test/features/practice4.3.feature (hunk 1 above)
Comment (JuanaRodriguez) [inline Juana/Cucumber_test/features/practice4.3.feature]: Yes, I forget

File: Juana/Cucumber_test/features/practice4.3.feature (hunk 2 above)
Comment (JuanaRodriguez) [inline Juana/Cucumber_test/features/practice4.3.feature]: OK thank you Karime

**Pull request merged**